samples/sensor/sensor_shell/prj.conf

```diff
@@ -4,5 +4,9 @@ CONFIG_SENSOR=y
 CONFIG_SENSOR_SHELL=y
 CONFIG_SENSOR_INFO=y
 
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_SHELL=y
+CONFIG_BQ274XX_PM=n
```

Comment:
why ? This is specific to one driver and this is a generic sample.

Reply:
The fact that the symbol exists at all is concerning to me

Reply:
yep